pkg/ccl/streamingccl/replicationtestutils/span_config_helpers.go

```diff
@@ -42,3 +53,67 @@ func RecordToEntry(record spanconfig.Record) roachpb.SpanConfigEntry {
 		Config: c,
 	}
 }
+
+// PrettyRecords pretty prints the span config target and config ttl.
+func PrettyRecords(records []spanconfig.Record) string {
+	var b strings.Builder
+	for _, update := range records {
+		b.WriteString(fmt.Sprintf(" %s: ttl %d,", update.GetTarget().GetSpan(), update.GetConfig().GCPolicy.TTLSeconds))
```

Comment:
It doesn't matter at all, but just so you know, you can do `fmt.Fprintf(&b, ...)`

Reply:
:D fmt.Fprintf takes a Writer and strings.Builder is a Writer per its public API: https://pkg.go.dev/strings#Builder.Write

Anyway, don't change this. Next time I'll not let the intrusive thoughts convince me to write this comment.